tests/unit/s2n_alerts_protocol_test.c

```diff
@@ -522,5 +529,110 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_shutdown(sender, &blocked));
     };
 
+    /* Test: End-of-Data
+     *
+     *= https://tools.ietf.org/rfc/rfc8446#6.1
```

Comment:
nit: These should have `type=test` I think?
```suggestion
     *= https://tools.ietf.org/rfc/rfc8446#6.1
     *= type=test
```

Reply:
Thanks, I forget that a lot 😅